bcdaworker/main.go

```diff
@@ -189,7 +191,10 @@ func appendErrorToFile(acoID, code, detailsCode, detailsDisplay string, jobID st
 
 	dataDir := os.Getenv("FHIR_STAGING_DIR")
 	fileName := fmt.Sprintf("%s/%s/%s-error.ndjson", dataDir, jobID, acoID)
-	f, err := os.OpenFile(fileName, os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0600)
+
+	/* #nosec -- 0640 permissions required for Splunk ingestion */
+	f, err := os.OpenFile(fileName, os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0640)
```

Comment:
Don't think this change is necessary since this is not a file that is ingested by Splunk.